tools/go.mod

```diff
@@ -78,28 +76,30 @@ require (
 	github.com/charithe/durationcheck v0.0.10 // indirect
 	github.com/chavacava/garif v0.1.0 // indirect
 	github.com/ckaznocha/intrange v0.2.0 // indirect
-	github.com/compose-spec/compose-go/v2 v2.1.6 // indirect
+	github.com/compose-spec/compose-go/v2 v2.4.7 // indirect
```

Comment:
This also fixes https://github.com/advisories/GHSA-36gq-35j3-p9r9, which just started showing up in scanning.